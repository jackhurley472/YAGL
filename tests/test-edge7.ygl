Graph g;

Node n("h");
Node n2("e");
Node n3("l");

int b = 6;

/* Must use [ ] for bidirectional with
   separate weights in recursion*/
g: + n + n2 + n3 [7<->5] n2;

/* [ ] optional for bidir with separate 
   weights with no recursion*/
g: n2 6<->9 n;
g: n |b+10|<->b n3;

printGraph(g);
