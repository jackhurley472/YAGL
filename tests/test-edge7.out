============== Graph Print ===============
	Stats: 	4	3	3

Nodes:
Node (0) : h --- Node (1) : e --- Node (2) : l


Edges:
[(2) : l] ---(5)---> [(1) : e]
[(2) : l] ---(16)---> [(0) : h]
[(1) : e] ---(7)---> [(2) : l]
[(1) : e] ---(9)---> [(0) : h]
[(0) : h] ---(6)---> [(1) : e]
[(0) : h] ---(6)---> [(2) : l]
============ End Graph Print =============
